examples/v2beta1/horovod/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN if [[ "${PYTHON_VERSION}" == "3.6" ]]; then \
     fi
 RUN ln -s /usr/bin/python${PYTHON_VERSION} /usr/bin/python
 
-RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
+RUN curl -O https://bootstrap.pypa.io/pip/3.6/get-pip.py && \
```

Comment:
This Dockerfile is based on ubuntu 18.04, which has EoL by the end of this month https://ubuntu.com/blog/18-04-end-of-standard-support

would you mind upgrading it to 22.04?

Reply:
I'm willing to, but I don't have the best practices for handling the version matrix (Python/TensorFlow/PyTorch/etc.). Please expect longer lead times.

Reply:
Thanks

let's merge this for now then
/lgtm
/approve